documentation/MeetingMinutes/MOM-2023-06-16.md

```diff
@@ -0,0 +1,110 @@
+    * Process how demand from product perspective can come into the scope definition of next release to be clarified
+    * Jose will invite TEF colleague into next call to introduce the Product team
+    * Objective of the team is to ensure that defined APIs will be implemented by as many operators as possible with the same coverage of the API features
+    * Consensus that the ratio between value for developers/API consumers vs implementation effert for operators is an important criteria for the decisions which enhancements should go into the next release
+    * Beside "product requirements" there could be technical requirements which need to be addressed
+
+* General notion that the next release should have a focus on stability and should avoid breaking changes
+
+* Discussion of the issues currently within backlog
+
+  * [#7 Add new endpoint in QoD API to request available qos_profiles](https://github.com/camaraproject/QualityOnDemand/issues/7)
+    * Base functionality done in v0.9.0, remaining functionality in #147 and #166
+    * Original issue to be closed<br>
+  * [#31 Ensure secure handling of callback URL](https://github.com/camaraproject/QualityOnDemand/pull/31)
+    * Security issue potentially not completely addressed within v0.9.0, Akos will create a new issue with the remaining points
+  * [#45 Need to support ability to configure fallback qos](https://github.com/camaraproject/QualityOnDemand/pull/45)
+    * Functionality of NEF in Rel 16/17, unclear if implementions within operator cores are available
+    * Product demand to be clarified
+  * [#51 Proposal of IpFlowDescription to allow asymmetric QoS flows and (re)introduce protocol definitions](https://github.com/camaraproject/QualityOnDemand/issues/51)
+    * Demand and implementation effort to be clarified
+  * [#101 List endpoint for active sessions of authenticated user](https://github.com/camaraproject/QualityOnDemand/pull/101)
+    * Requests from developers known (to recover session IDs)
+    * Implementation be possible based on clientId used in OAuth authentication
+  * [#147 Extend the query capabilities for Qos Profiles](https://github.com/camaraproject/QualityOnDemand/pull/147)
+    * Derived from [#7](https://github.com/camaraproject/QualityOnDemand/issues/7) above
+    * Demand and implementation effort to be clarified
+  * [#166 Extend QoS Profile queries to list profiles based on specific users or devices](https://github.com/camaraproject/QualityOnDemand/pull/166)
+    * Derived from [#7](https://github.com/camaraproject/QualityOnDemand/issues/7) above
+    * Demand most likely, implementation effort to be clarified
+
+### Issues closed / to be closed:
+
+  * Issue #144, fixed by #151
+  * Issue #7, remaining issues within #147 and #166
+
+## AOB
+
+* Topics for next call
+  * Status of release (candidate) v0.9.0
+  * PR for #158
+  * Discussion of product team request for next release
+* The next call will be on June 16th, 2 PM CEST / 12:00 UTC
```

Comment:
Next meeting date is incorrect. Should be June 30th.

Reply:
Thanks ... #170